pkgs/by-name/se/seagoat/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+  python3Packages,
+  ripgrep,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "seagoat";
+  version = "0.48.4";
+  pyproject = true;
+
```

```suggestion
  pythonRelaxDeps = [
    "setuptools"
  ];
```
Does not build otherwise.

---

Thanks a lot.